tests/vegalite/v5/test_api.py

```diff
@@ -1607,51 +1607,72 @@ def test_polars_with_pandas_nor_pyarrow(monkeypatch: pytest.MonkeyPatch):
     assert "numpy" not in sys.modules
 
 
-@pytest.mark.skipif(
-    Version("1.5") > PANDAS_VERSION,
-    reason="A warning is thrown on old pandas versions",
-)
-@pytest.mark.xfail(
-    sys.platform == "win32", reason="Timezone database is not installed on Windows"
-)
-def test_ibis_with_date_32():
-    ibis.set_backend("polars")
-    df = pl.DataFrame(
+@skip_requires_pyarrow(requires_tzdata=True)
+def test_interchange_with_date_32():
+    # Test that objects which Narwhals only supports at the interchange
+    # level can be plotted when they contain date32 columns.
+    df = pl.DataFrame(  # noqa: F841
         {"a": [1, 2, 3], "b": [date(2020, 1, 1), date(2020, 1, 2), date(2020, 1, 3)]}
     )
-    tbl = ibis.memtable(df)
-    result = alt.Chart(tbl).mark_line().encode(x="a", y="b").to_dict()
+    rel = duckdb.sql("select * from df")
+    result = alt.Chart(rel).mark_line().encode(x="a", y="b").to_dict()
     assert next(iter(result["datasets"].values())) == [
         {"a": 1, "b": "2020-01-01T00:00:00"},
         {"a": 2, "b": "2020-01-02T00:00:00"},
         {"a": 3, "b": "2020-01-03T00:00:00"},
     ]
 
 
-@pytest.mark.skipif(
-    Version("1.5") > PANDAS_VERSION,
-    reason="A warning is thrown on old pandas versions",
-)
-@pytest.mark.xfail(
-    sys.platform == "win32", reason="Timezone database is not installed on Windows"
-)
-def test_ibis_with_vegafusion(monkeypatch: pytest.MonkeyPatch):
```

Comment:
@MarcoGorelli I'm assuming dropping this was unintentional?

In (https://github.com/vega/altair/pull/3672/commits/c32e1eb625919a46e4b1a3b04e257cb279b83330) I've moved some things around to make this harder to miss in the future.

Reply:
As in, the skipif? I was just curious to see if it was an ibis-specific thing and was still needed